rootfs/Dockerfile

```diff
@@ -3,13 +3,14 @@ FROM gcr.io/google_containers/ubuntu-slim:0.4
 ENV TERM=xterm
 
 # disable source repos (speeds up apt-get update)
-RUN sed -i -e 's/^deb-src/#deb-src/' /etc/apt/sources.list && \
-apt-get update && \
-apt-get install -y \
-	bash \
-	curl \
-	net-tools \
-	procps && \
-	apt-get clean && \
-	rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/* /usr/share/man /usr/share/doc && \
-bash -c "mkdir -p /usr/share/man/man{1..8}"
+RUN sed -i -e 's/^deb-src/#deb-src/' /etc/apt/sources.list \
+	&& apt-get update \
+	&& apt-get upgrade -y \
+	&& apt-get install -y \
```

Comment:
Seems like it isn't being respected apparently. Also I thought it was a bit sloppy to put `ENV DEBIAN_FRONTEND` in the Dockerfile and thus the overall container environment, but whatever works is fine (including ignoring my comment--it's just cosmetic).